rest_api/config_apis/image-config-openshift-io-v1.adoc

```diff
@@ -178,11 +178,11 @@ Type::
 
 | `externalRegistryHostnames`
 | `array (string)`
-| externalRegistryHostnames provides the hostnames for the default external image registry. The external hostname should be set only when the image registry is exposed externally. The first value is used in 'publicDockerImageRepository' field in ImageStreams. The value must be in "hostname[:port]" format.
+| `externalRegistryHostnames` provides the hostnames for the default external image registry. The external hostname should be set only when the image registry is exposed externally. The first value is used in `publicDockerImageRepository` field in image streams. The value must be in `hostname[:port]`` format.
```

Comment:
You have two back ticks after "hostname[:port]". Should this be only one back tick?

Reply:
Yep, good catch, thank you!